### DISCUSSION TITLE
Question about barMinHeight in v7

### DISCUSSION THREAD

**Discussion by euvl** [katspaugh/wavesurfer.js#2815] [General]
Hi there! V7 looks great and works really well! One questions i had is about the strategy around barMinHeight.

On v6 my component looked like this:
 
<img width="522" alt="Screenshot 2023-05-31 at 10 26 28 pm" src="https://github.com/katspaugh/wavesurfer.js/assets/1577802/cda300b0-83fe-4d8b-bb43-25ab8f58bcf1">

With `barMinHeight` being `2px` - it was creating a "line" during the moments of silence.

In v7 same component looks like this:

<img width="516" alt="Screenshot 2023-05-31 at 10 26 36 pm" src="https://github.com/katspaugh/wavesurfer.js/assets/1577802/3b728f83-a9d4-48e2-93d4-fdc2c0841ccc">

And I am a little puzzled to how can i achieve same effect as the property is gone.

If you have any ideas please let me know! Thanks!!

**Comment by katspaugh** [comment]
I suggest we make it always render at least one pixel. I’m currently refactoring the rendering code a bit, I’ll make sure this is fixed along the way. 
Update: should be fixed here: https://github.com/katspaugh/wavesurfer.js/pull/2819

**Reply by euvl** [reply]
Amazing, thank you so much 👍

**Reply by katspaugh** [reply]
Just to clarify, I'm not suggesting that you should write a custom render function, I'm just saying I fixed the min bar height. It should always render at least a one-pixel bar now.

**Reply by dincozdemir** [reply]
@katspaugh Should we write a custom render function to set a minimum bar height now?

**Reply by katspaugh** [reply]
If you want it to be more than 1px – yes, or you can make a PR to add this parameter back.
